src/dra/63ED4.c

```diff
@@ -602,7 +602,105 @@ void func_8010427C(void) {
     FreePrimitives(D_80137E48);
 }
 
-INCLUDE_ASM("dra/nonmatchings/63ED4", func_801042C4);
+void func_801042C4(s32 arg0) {
+    VECTOR sp10;
+    Primitive* prim;
+    s32 i;
+
+    D_80137EE0 = arg0;
+    D_80097C98 = 0;
+    func_800EA5E4(0x18U);
```

Comment:
```suggestion
    func_800EA5E4(0x18);
```
Let's remove the `U`, they are almost always unnecessary and very unlikely used by the original developers